student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -125,11 +136,11 @@ private static boolean isClientError(HttpStatus status) {
     
     try {
       responseEntity = restTemplate.exchange(
-          builder.build().encode().toUri(),
```

Comment:
My apologies for these accidental formatting commits - but on that note, are we using 2-space indents or 4-space indents for new lines in legacy TDS_Student?